libs/domain/user/account-menu/account-menu.component.ts

```diff
@@ -0,0 +1,43 @@
+import { ContextController } from '@spryker-oryx/core';
+import { ButtonType } from '@spryker-oryx/ui/button';
+import { IconTypes } from '@spryker-oryx/ui/icon';
+import { AccountMenuLinks } from '@spryker-oryx/user';
+import {
+  I18nMixin,
+  hydrate,
+  signal,
+  signalAware,
+} from '@spryker-oryx/utilities';
+import { LitElement, TemplateResult, html } from 'lit';
+import { repeat } from 'lit/directives/repeat.js';
+import { styles } from './account-menu.styles';
+
+@signalAware()
+@hydrate({ event: 'window:load' })
+export class AccountMenuComponent extends I18nMixin(LitElement) {
+  static styles = styles;
+  protected links = [
+    { id: AccountMenuLinks.OVERVIEW, icon: IconTypes.Description },
+    { id: AccountMenuLinks.PROFILE, icon: IconTypes.User },
+    { id: AccountMenuLinks.ADDRESSES, icon: IconTypes.Location },
+    { id: AccountMenuLinks.ORDERS_HISTORY, icon: IconTypes.OrdersHistory },
+  ];
+
+  protected contextController = new ContextController(this);
+
+  protected $tab = signal(this.contextController.get<string>('tab'));
```

Comment:
Why do we need to store the tab (why is it tab, btw?) in the context? Each link has different href, I believe that we can just check it and decide which one is active